react-components/src/components/RevealContainer/RevealContainer.tsx

```diff
@@ -1,43 +1,55 @@
-import { CogniteClient } from "@cognite/sdk";
-import { useEffect, useRef, ReactNode, useState, useMemo } from "react";
+/*!
+ * Copyright 2023 Cognite AS
+ */
+import { type CogniteClient } from '@cognite/sdk';
+import { useEffect, useRef, type ReactNode, useState, type ReactElement } from 'react';
 import { Cognite3DViewer } from '@cognite/reveal';
-import { RevealContext } from "./RevealContext";
-import { Color } from "three";
+import { RevealContext } from './RevealContext';
+import { type Color } from 'three';
 
-type RevealContainerProps = {
+interface RevealContainerProps {
   color?: Color;
```

Comment:
Will the lint rule prefer `interface` over `type`, or is this change just a matter of taste?